manage.py

```diff
@@ -6,6 +6,7 @@
 
 def main():
     """Run administrative tasks."""
+
```

Comment:
Delete line between comment and code

Reply:
```suggestion

```

Reply:
Unresolved as line is not deleted

Reply:
locally delete line and push changes to pull request

Reply:
Make changes in this branch

Reply:
![image](https://github.com/Strexas/QrWeb/assets/83172770/e3b3bcc8-fca5-4cab-83c9-7cb256679ced)

Reply:
Locally make changes, commit and push commit to this branch

Reply:
![image](https://github.com/Strexas/QrWeb/assets/83172770/d75c2d6c-7fd8-43ad-9ba1-267dd11b582a)
??

Reply:
Give me output of
```
git status
git push
```

Reply:
![image](https://github.com/Strexas/QrWeb/assets/83172770/038df204-86c9-4418-9886-0d432184f689)

Reply:
![image](https://github.com/Strexas/QrWeb/assets/83172770/30e058b6-d779-4018-bc79-3450bf1d2748)

Reply:
![image](https://github.com/Strexas/QrWeb/assets/83172770/a1ab9778-6b56-4625-b961-9c5ac5978e12)

Reply:
Try again, if there still error, we will today resolve it together